README.md

```diff
@@ -64,7 +64,11 @@ $facade->getReservationsByInterval(\Carbon\Carbon $from, \Carbon\Carbon $to);
 $facade->isDateAvailable(\Carbon\Carbon $date);
 ```
 
-## Override configurations
+## Configuration
+
+You can find plugin configuration at the CMS backend. But you can also set some values at plugin's config file. Config values 
```

Comment:
I move only resarvation and date format to settings. Mail, random, and other don't move to Settings.